core/pta/remote_attestation/base64.c

```diff
@@ -0,0 +1,58 @@
+#include "base64.h"
+
+static const char base64_table[] =
+    "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/";
+
+size_t base64_enc_len(size_t size) { return 4 * ((size + 2) / 3) + 1; }
+
+/**
+   base64 Encode a buffer (NUL terminated)
+   @param in      The input buffer to encode
+   @param inlen   The length of the input buffer
+   @param out     [out] The destination of the base64 encoded in
+   @param outlen  [in/out] The max size and resulting size
+   @return 1 if successful
+*/
+int base64_encode(const unsigned char *in, unsigned long inlen, char *out,
```

Comment:
This file looks like a copy & paste from `lib/libutee/base64.c`. Instead, we should make the needed functions available to be called from this PTA. It could be done in this PR, but I think doing that first in another PR would be easier. Anyway, the base64 routines should be moved into `lib/libmisc` while making the functions publicly callable.